contracts/interfaces/draft-IERC6093.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.19;
 /**
  * @dev Standard ERC20 Errors
  * Interface of the ERC6093 custom errors for ERC20 tokens
- * as defined in https://eips.ethereum.org/EIPS/eip-6093
+ * as defined in https://eips.ethereum.org/EIPS/eip-6093[the EIP]
```

Comment:
I'd write it as 
```
* Interface of the https://eips.ethereum.org/EIPS/eip-6093[ERC-6093] custom errors for ERC20 tokens
```

Reply:
I agree, extending it was too repetitive, and didn't notice. Updated